solidus_volume_pricing.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_runtime_dependency 'solidus_core', '~> 1.3'
+  s.add_runtime_dependency 'solidus_core', '~> 2.1'
```

Comment:
All tests now pass and it works with anything Solidus 1.3 and up. Anything else I need to do to get this up to snuff?